lib/mallet.js

```diff
@@ -124,6 +125,120 @@ class Iele {
     tx.data = '0x' + rlp.encode([code, args]).toString('hex');
     return this.mallet.sendTransaction(tx, password);
   }
+
+  compileSol(source) {
```

Comment:
maybe `compileSolToIele` to make it clear what it does?

Reply:
Well, it is part `Iele` class, besides it's not really meant to be called by users. I will add `_` to those names.

Reply:
ok, nvm I didn't notice it's inside Iele class, I thought it's directly under mallet.js file